pkg/services/accesscontrol/models.go

```diff
@@ -341,16 +341,40 @@ const (
 
 	// Dashboard scopes
 	ScopeDashboardsAll = "dashboards:*"
+
+	// Alerting rules actions
+	ActionAlertingRuleCreate = "alert.rules:create"
+	ActionAlertingRuleRead   = "alert.rules:read"
+	ActionAlertingRuleUpdate = "alert.rules:update"
+	ActionAlertingRuleDelete = "alert.rules:delete"
+
+	// Alerting instances (+silences) actions
+	ActionAlertingInstanceCreate = "alert.instances:create"
+	ActionAlertingInstanceUpdate = "alert.instances:update"
+	ActionAlertingInstanceRead   = "alert.instances:read"
+
+	// Alerting Notification policies actions
+	ActionAlertingNotificationsCreate = "alert.notifications:create"
+	ActionAlertingNotificationsRead   = "alert.notifications:read"
+	ActionAlertingNotificationsUpdate = "alert.notifications:update"
+	ActionAlertingNotificationsDelete = "alert.notifications:delete"
+
+	// External alerting rule actions
+	ActionAlertingRuleExternalEdit = "alert.rules.external:edit"
+	ActionAlertingRuleExternalRead = "alert.rules.external:read"
+
+	// External alerting instances actions
+	ActionAlertingInstancesExternalEdit = "alert.instances.external:edit"
+	ActionAlertingInstancesExternalRead = "alert.instances.external:read"
+
+	// External alerting notifications actions
+	ActionAlertingNotificationsExternalEdit = "alert.notifications.external:edit"
+	ActionAlertingNotificationsExternalRead = "alert.notifications.external:read"
```

Comment:
I had to move action declaration from alerting package to avoid cycle dependencies (in another PRs)